src/source/video_source.js

```diff
@@ -79,6 +79,9 @@ class VideoSource extends ImageSource {
                 this.video = video;
                 this.video.loop = true;
 
+                // Prevent the video from taking over the screen in iOS
+                this.video.setAttribute('playsinline', '');
```

Comment:
Going to check this when on a better wifi network, and if it works, 🔪  a few characters.

```suggestion
                this.video.playsinline = '';
```

Reply:
Doesn't work 🤷